src/appleseed-max-impl/appleseedrenderer/resource.h

```diff
@@ -52,14 +79,13 @@
 #define IDC_STATIC_SCALE_MULTIPLIER                 407
 #define IDC_TEXT_SCALE_MULTIPLIER                   408
 #define IDC_SPINNER_SCALE_MULTIPLIER                409
-
 #define IDD_FORMVIEW_RENDERERPARAMS_SYSTEM          500
 #define IDC_TEXT_RENDERINGTHREADS                   501
 #define IDC_SPINNER_RENDERINGTHREADS                502
 #define IDC_CHECK_LOW_PRIORITY_MODE                 503
 #define IDC_CHECK_USE_MAX_PROCEDURAL_MAPS           504
 #define IDC_CHECK_LOG_MATERIAL_EDITOR               505
-
+#define IDC_CHECK_LOW_PRIORITY_MODE2                506
```

Comment:
Why do we both have `IDC_CHECK_LOW_PRIORITY_MODE` and `IDC_CHECK_LOW_PRIORITY_MODE2`?

Reply:
This comes when one copies and pastes an UI-element in the resource view, e.g. a spinner element for a new parameter. VS Studio then automatically adds a corresponding entry for it in resource.h. When the UI-element name is then updated to fit the new parameter the old entry in resource.h still remains and can be safely deleted.  
I deleted the entry.